packages/code-du-travail-frontend/src/web-components/tooltip-cc.js

```diff
@@ -0,0 +1,119 @@
+import { LitElement, html, css } from "lit-element";
+import { v4 as uuidv4 } from "uuid";
+import { throttledDisplayInViewport } from "./utils";
+
+class WebComponentsTooltipCC extends LitElement {
+  static get properties() {
+    return {
+      id: { type: String },
+      visible: { type: Boolean },
+    };
+  }
+
+  constructor() {
+    super();
+    this.id = uuidv4();
```

Comment:
why do we need this ?

Reply:
pour le positionnement du tooltip c'est bon, j'ai tout simplifié, c'est plus simple que ce qui est fait sur reach
pour l'uuid c'est parce que j'ai choisi de tout mettre dans le dom dès le début avec du describedby comme ici https://van11y.net/downloads/simple-tooltip/demo/index-fr.html

Reply:
bien vu pour le transpileModule ! much better !